localsettings.example.py

```diff
@@ -230,54 +108,3 @@
                                   # to 200 to prevent AssertionError: 429 != 200
                                   # test failures in development environsments.
 CCHQ_API_THROTTLE_TIMEFRAME = 10  # seconds
```

Comment:
these api settings feel like they could be in dev settings

Reply:
agreed, wanted to make sure they were the right ones first: https://github.com/dimagi/commcare-hq/pull/29626/commits/d47f9f45ba524c5e7286999faedeaa35aba6d471